.travis.yml

```diff
@@ -55,7 +55,8 @@ jobs:
     - WEB3_REQUIRED_NETWORK_ID=1
     - WEB3_PUBLIC_HTTP_PROVIDER=https://mainnet.infura.io/v3/17c3985baecb4c4d94a1edc2c4d23206
     - WEB3_PUBLIC_WS_PROVIDER=wss://mainnet.infura.io/ws/v3/17c3985baecb4c4d94a1edc2c4d23206
-    - SENTRY_DSN=https://e012761929d24baa96ca82772f60b395@sentry.io/1203154
+    - SENTRY_DSN=https://e012761929d24baa96ca82772f60b395@o151964.ingest.sentry.io/1203154
+    - SENTRY_INDEXER_DSN=https://8311f8e7df9046b781600f95eefd1aa0@o151964.ingest.sentry.io/5235991
```

Comment:
@fonty1 Ended up with a separate variable, it won't be enabled by defualt locally but defined in `travis.yml` so should work in prod and push to the indexer Sentry project